pkg/apis/core/v1beta1/validation/validation_test.go

```diff
@@ -288,3 +292,111 @@ func federatedTypeConfig(apiResource *metav1.APIResource) *v1beta1.FederatedType
 	}
 	return ftc
 }
+
+func TestValidateKubeFedConfig(t *testing.T) {
+	errs := ValidateKubeFedConfig(validKubeFedConfig())
+	if len(errs) != 0 {
+		t.Errorf("expected success: %v", errs)
+	}
+
+	errorCases := map[string]*v1beta1.KubeFedConfig{}
+
+	validScope := validKubeFedConfig()
```

Comment:
Please order these in the same order that they appear in the code (which is itself ordered as per the type definition) to simplify future maintenance.